CHANGELOG.md

```diff
@@ -25,6 +25,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
     also accept `--tuning` and `--size` options (although there are
     not yet any tunable sizes).
 
+  * New functions: `soacs.scatter_2d` and `soacs.scatter_3d` for scattering to
+    multi-dimensional arrays (#1258).
```

Comment:
Why the `soacs.` prefix?

Reply:
I was just copying the syntax from the notice about `f32.recip` and `f64.recip` from above. I can remove it if you want.

Reply:
`f32` and `f64` are actual modules, so those are the names of the functions.  There is no `soacs` module (but one could argue that perhaps there should be).

Reply:
Oh, of course. I'll remove it for now.